models/auth/oauth2_grant.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
```

Comment:
Is this "code moving" related to the `db.FInd`? If it is not related, it just makes the PR more complex than it should be and may introduce new bugs.

Reply:
Done.